spec/system/page_objects/pages/search.rb

```diff
@@ -103,6 +110,10 @@ def active?
       def not_active?
         has_no_css?(SEARCH_PAGE_SELECTOR)
       end
+
+      def ctrl_f_search
```

Comment:
Maybe could be `keyboard_shortcut_search`, since it's not Ctrl on Mac. 🙂

Reply:
Yah true, I thought it was a bit dumb when naming it and now you confirmed :joy: